src/components/ResourceList/components/FilterControl/components/DateSelector/DateSelector.tsx

```diff
@@ -408,8 +413,19 @@ function getDateFilterOption(
   return filterValue;
 }
 
-function formatDateValue(date: Date) {
-  return date.toISOString().slice(0, 10);
+function formatDateForLocalTimezone(date: Date) {
+  const timezoneOffsetMs = date.getTimezoneOffset() * 60 * 1000;
+  const formattedDate = new Date();
+
+  if (date.getTimezoneOffset() > 0) {
+    const newTime = date.getTime() + timezoneOffsetMs;
+    formattedDate.setTime(newTime);
+    return formattedDate.toISOString();
+  }
+
+  const newTime = date.getTime() - timezoneOffsetMs;
```

Comment:
I don't think that's possible unfortunately, since when it's a negative timezone offset we want to _subtract_ and when it's a positive we want to _add_.